packages/core/src/components/context-menu/contextMenuTarget.tsx

```diff
@@ -42,7 +43,11 @@ export function ContextMenuTarget<T extends { prototype: IContextMenuTarget }>(c
             const menu = this.renderContextMenu(e);
             if (menu != null) {
                 e.preventDefault();
-                ContextMenu.show(menu, { left: e.clientX, top: e.clientY });
+                ContextMenu.show(menu, { left: e.clientX, top: e.clientY }, () => {
+                    if (isFunction(onCloseContextMenu)) {
+                        onCloseContextMenu.call(this);
```

Comment:
we actually have a utility function for this pattern, called `safeInvoke`. [usage example](https://github.com/palantir/blueprint/blob/a5f1b24a444e8523e00b12ff6d6c105147f844fb/packages/core/src/components/toast/toast.tsx#L109)

but i think you can get even smarter and just pass the param directly as the third argument, since `ContextMenu` [already uses `safeInvoke` for this option](https://github.com/palantir/blueprint/blob/master/packages/core/src/components/context-menu/contextMenu.tsx#L66).

```js
ContextMenu.show(menu, { left: e.clientX, top: e.clientY }, onCloseContextMenu);
```